modules/nw-ingress-setting-update-subnet-loadbalancerservice.adoc

```diff
@@ -0,0 +1,83 @@
+// Modules included in the following assemblies:
+//
+// * ingress/configure-ingress-operator.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-ingress-setting-update-subnet-Loadbalancerservice_{context}"]
+= Updating the subnets on an existing Ingress Controller
++
+You can update an `IngressController` with manually specified load balancer subnets in {product-title} to avoid any disruptions, to maintain the stability of your services, and to ensure that your network configuration aligns with your specific requirements. The following procedures show you how to select and apply new subnets, verify the configuration changes, and confirm successful load balancer provisioning. 
+
+.Procedure
+To update an `IngressController` with manually specified load balancer subnets, you can follow these steps:
+
+. Choose the subnets.
++
+[source,yaml]
+----
+apiVersion: operator.openshift.io/v1
+kind: IngressController
+metadata:
+  name:  <name> <1>
+  namespace: openshift-ingress-operator
+spec:
+  domain: <domain> <2>
+  endpointPublishingStrategy:
+    type: LoadBalancerService
+    loadBalancer:
+      scope: External
+      providerParameters:
+        type: AWS
+        aws:
+          type: Classic <3>
+          classicLoadBalancer: <4>
+            subnets:
+              ids: <5>
+              - <updated_subnet> <6>
+              - <updated_subnet>
+              - <updated_subnet>
+----
+<1> Replace `<name>` with a name for the `IngressController`.
+<2> Replace `<domain>` with the DNS name serviced by the `IngressController`.
+<3> Specify updated subnet IDs (or names if you using `names`).
+<4> You can also use the `networkLoadBalancer` field if using an NLB.
+<5> You can optionally specify a subnet by name using the `names` field instead of specifying the subnet by ID.
+<6> Specify subnet IDs (or names if you using `names`).
+
```

Comment:
What do you think about also adding the same warning about subnet restrictions here? Not sure if docs like each procedure to stand independently (assuming they didn't read the other procedure), your call on this:

 ```suggestion
[IMPORTANT]
====
You can specify a maximum of one subnet per availability zone. Only provide public subnets for external Ingress Controllers and private subnets for internal Ingress Controllers.
====
```